File: CoffeeShopProject/Models/HoaDon.cs
using System;
using System.Collections.Generic;

namespace CoffeeShopProject.Models
{
    public partial class HoaDon
    {
        public int MaHoaDon { get; set; }
        public DateTime? ThoiGianLap { get; set; }
        public int? MaNhanVienOrder { get; set; }
        public int? MaBan { get; set; }
        public double? TongTien { get; set; }
        public string MaHoaDonLocal { get; set; }
        public int? TrangThai { get; set; }
        public int? GiamGia { get; set; }
        public double? ThanhTien { get; set; }
        public int? MaThuNgan { get; set; }
    }
}

File: CoffeeShopDATN_Android/app/src/main/java/fpo/thanh/coffeeshop/domain/Room/RoomModel/HoaDon.java
package fpo.thanh.coffeeshop.domain.Room.RoomModel;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import fpo.thanh.coffeeshop.shareModel.DsMonModel;

@Entity
public class HoaDon {
    @PrimaryKey(autoGenerate = false)
    public Integer maHoaDon;
    @ColumnInfo(name = "thoiGianLap")
    public String thoiGianLap;
    @ColumnInfo(name = "maNhanVien")
    public Integer maNhanVien;
    @ColumnInfo(name = "maBan")
    public Integer maBan;
    @ColumnInfo(name = "tongTien")
    public Integer tongTien;
    @ColumnInfo(name = "maHoaDonLocal")
    public Integer maHoaDonLocal;
    @ColumnInfo(name = "trangThai")
    public String trangThai;

}
